src/common/evaluation/QA/eval_depthmap_models/src/qa_config_height_deep_ensemble.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from bunch import Bunch
+
+CONFIG_NAME = os.path.splitext(os.path.basename(__file__))[0]
+
+# Details of model used for evaluation
+MODEL_CONFIG = Bunch(dict(
+    EXPERIMENT_NAME='q1-ensemble-warmup',
+    RUN_ID='all',
```

Comment:
I suggest to specify all run ids. You can find them in the AzureML experiments

```suggestion
    RUN_IDS=['q1-ensemble-warmup_1610544610_eb44bfe2', 'q1-ensemble-warmup_1610547587_7ca932c3', ...],
```